pom.xml

```diff
@@ -67,6 +67,11 @@
       <version>2.4.0</version>
     </dependency>
     <dependency>
+      <groupId>org.jenkins-ci.plugins</groupId>
+      <artifactId>subversion</artifactId>
+      <version>2.5</version>
```

Comment:
I would vote against adding it as a mandatory dependency since not everybody uses it now.
Maybe Git should be converted to a optional dependency as well

In order to run tests, just add 
```
<scope>test</scope>
```